msteams-platform/m365-apps/extend-m365-teams-personal-tab.md

```diff
@@ -233,6 +250,12 @@ Upon first run of local debug to Office or Outlook, you'll be prompted to sign i
 
 Provide feedback and report any issues with the Teams Toolkit debugging experience at [Microsoft Teams Framework (TeamsFx)](https://github.com/OfficeDev/TeamsFx/issues).
 
+#### Mobile debugging
+
+1. You can remotely debug your app in Office app for Android. From your profile **(Me) > Settings > Enable remote debugging**, toggle on the option for **Enable remote debugging**.  In Office mobile Android, Enable remote debugging tool in Settings lets you see all sideloaded apps in the Apps menu.
+1. From your dev machine, open your browser to the DevTools inspection page. For example ,go to `edge://inspect/#devices` in Microsoft Edge to display a list of debug-enabled Android WebViews.
```

Comment:
We need to launch the app from start after enabling remote debugging and then only it will end up in edge://inspect/#devices tab.

Reply:
> We need to launch the app from start after enabling remote debugging and then only it will end up in edge://inspect/#devices tab.